app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -115,6 +75,47 @@ class MySpacesSection extends ConsumerWidget {
       ),
     );
   }
+
+  Widget? _spaceBuilder(
```

Comment:
Widget method patterns aren't performance friendly, so try to have it minimised whenever possible and instead extract widgets or use stateless widget patterns.